src/sincere_singularities/modules/restaurants_view.py

```diff
@@ -99,6 +103,8 @@ async def _stop_game(self, *_: disnake.ui.Button | disnake.MessageInteraction) -
 class Restaurants:
     """Class to Manage the Restaurants & UI"""
 
+    condition_manager: "ConditionManager"
```

Comment:
Why isn't this part of `__init__`?

Reply:
This isnt possible, because ConditionManager needs the Restaurants obj in its __init__ see [bot.py](https://github.com/SincereSingularities/SincereSingularities/blob/586079d150e6247c015771b41cbb6a648796082a/src/sincere_singularities/bot.py#L77).
This would mean we would have circular __init__ arguments.